MAINTAINERS.md

```diff
@@ -0,0 +1,8 @@
+Repository Maintainers
+======================
+
+See the information about [community membership roles](https://wiki.lfedge.org/display/OH/Community+Membership) to learn about the role of the maintainers and the process to become one.
+
+| Name         | GitHub      | email                    |
+| ------------ | ----------- | ------------------------ |
+| Joe Pearson  | joewxboy    | <joe.pearson@us.ibm.com> |
```

Comment:
You can add your name to this list, if you'd like, since you contributed these files.